newdle/client/src/components/CreateNewdle.js

```diff
@@ -93,3 +94,86 @@ function TimeSlotsPage() {
     </div>
   );
 }
+
+function FinalizePage() {
+  const [success, setSuccess] = useState(false);
+  const [newdle, setNewdle] = useState(null);
+  const title = useSelector(getTitle);
+  const duration = useSelector(getDuration);
+  const timeslots = useSelector(getFullTimeslots);
+  const timezone = 'UTC';
```

Comment:
while we'll eventually add a dropdown to select one, what about using `Europe/Zurich` for now?